src/pages/Login/Login.js

```diff
@@ -1,7 +1,83 @@
-import React from 'react';
+import React, { useState } from 'react';
+import AccountContainer from '../../components/AccountContainer/AccountContainer';
+import Button from '../../components/Button/Button';
+import Input from './components/Input';
+import styled from 'styled-components';
 
-function Login() {
-  return <></>;
+function Login(props) {
+  const [inputs, setInputs] = useState({
+    email: '',
+    password: '',
+  });
+  const { email, password } = inputs;
+  const isValid = email.includes('@', '.') && password.length >= 6;
+
+  const handleInput = e => {
+    const { name, value } = e.target;
+    console.log(name, value);
+    setInputs({
+      ...inputs,
+      [name]: value,
+    });
+  };
+
+  const handelLogin = (e, props) => {
+    e.preventDefault();
+    const { email, password } = inputs;
+    if (email.includes('@', '.') && password.length >= 6) {
+      fetch('http://10.58.2.241:8000/users/signup', {
+        method: 'POST',
+        body: JSON.stringify({
+          email: email,
+          password: password,
+        }),
+      })
```

Comment:
```suggestion
  const handelLogin = (e, props) => {
    e.preventDefault();
    
    if (isValid) {
      fetch(`${API}/users/signup`, {
        method: 'POST',
        body: JSON.stringify({ email, password }),
      })
```
- 외부에서 정의해둔 변수들을 사용하지 않고 있는 것 같습니다
- API 주소는 src/config.js에 분리해주세요
- body에 넘겨주고 있는 객체에서 key와 value의 변수명이 같을 경우 생략해줄 수 있습니다.

Reply:
넵!